src/main/java/com/courtcanva/ccfranchise/constants/OrderStatus.java

```diff
@@ -0,0 +1,6 @@
+package com.courtcanva.ccfranchise.constants;
+
+public enum OrderStatus {
+    UNASSIGNED,
```

Comment:
Think about the OrderStatus, if one order is sent to one franchisee, waiting for acceptance, what will be the status? how to prevent to send to other franchisees?

Reply:
The current order status is designed like this: UNASSIGNED, ASSIGNED_PENDING, ACCEPTED, COMPLETED, CANCELED
the meaning for each value is: UNASSIGNED // order created, website admin hasn't assigned to any franchisee ASSIGNED_PENDING // website admin assigned order to one franchisee, but he hasn't chosen accept or decline ACCEPTED // one franchisee accepted the order COMPLETED // franchisee completed the order CANCELED // order canceled by customer or website admin